android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildViewModel.kt

```diff
@@ -80,14 +82,31 @@ class HomeListChildViewModel @Inject constructor(
 				listingParams = getListingParams,
 				listingsResponse = runCatching {
 					runBlocking {
+						Log.d("lowerBound",getListingParams.locationRange.lowerBound?.toFloat().toString())
 						// TODO: Change to use filter values
 						listingsApi.listingsList(
-							priceMin = getListingParams.priceRange.lowerBound,
-							priceMax = getListingParams.priceRange.upperBound,
-							roomsMin = null,
-							roomsMax = null,
+//						TODO: ADDING LOCATION VALUE
+							longitude = 0f,
```

Comment:
LocationService is acting wired and crashing the app, so currently hardcoded the value, will update the location later in a separate PR.

Reply:
Ah, yeah, `LocationService` does that lol...
If you're talking about it crashing the entire app despite putting it in a different thread with a try catch around it that is.